pages/blog/dark-mode.mdx

```diff
@@ -0,0 +1,45 @@
+import withPost from '../../components/blog/WithPost'
+import Image from '../../components/Image'
+
+export const meta = {
+  title: 'Announcing Queue v1.2.0: Dark Mode & Access Tokens!',
+  date: '2019-04-17T18:00:00-0500',
+  featured: true,
+  authors: [
+    {
+      name: 'Jacqueline Osborn',
+      github: 'jackieo5023',
+      avatar: '/static/team/jackie.jpeg',
+    },
+    {
+      name: 'James Wang',
+      github: 'james9909',
+      avatar: '/static/team/james.jpg',
+    },
+  ],
+}
+
+export default withPost({ ...meta })
+
+## Dark Mode
+
+If you've logged into the queue recently, you may have noticed there is a new toggle near the logout button. Clicking this will enable our new highly sought after feature - dark mode! We hope being on the queue will no longer burn out your eyes (just your soul).
+
+<Image
+  src="/static/blog/dark-mode/dark-mode-view.jpg"
+  caption="Dark mode"
+  shadow
+/>
+
+## Access Tokens
+
+We've also rolled out access tokens, which can be managed through the user settings page. These tokens allow you to access the API as if you were logged in through the website. As such, you should be careful about keeping your tokens private. With an access token, you can:
```

Comment:
maybe add how to get to your user settings page (click on your name) - I've noticed some people don't know that page exists.